doc/fortunes.tips

```diff
@@ -28,7 +28,7 @@ Add comments using the ';' key in visual mode or the 'CC' command from the rizin
 Assemble opcodes with the 'a' and 'A' keys in visual mode, which are bindings to the 'wa' and 'wA' commands
 Find expanded AES keys in memory with '/ca'
 Find wide-char strings with the '/w <string>' command
-Enable ascii-art jump lines in disassembly by setting 'e asm.lines=true'. asm.lines.out and asm.linestyle may interest you as well
+Enable ascii-art jump lines in disassembly by setting 'e asm.lines=true'. asm.lines.out may interest you as well
```

Comment:
It is there by default. I'd remove this fortune completely

Reply:
Also, might be nor related to this PR ? :D

Reply:
Well, if you just used it for two changes then that's fine. But better to separate it haha

Reply:
>It is there by default. I'd remove this fortune completely

Oh, really? Sorry about that. Didn't realize that.
Yeah, just saw that it's turned on by default. 
Will remove that line 👍🏻 

> Also, might be nor related to this PR ? :D

I was about to ask you about that. Should that be another PR or not?

Reply:
usually, it should, but we can skip this time I think :slightly_smiling_face: It's not a big deal

Reply:
As discussed, I'll send another PR to fix this.

Reply:
i agree. this should go.

Reply:
Removing this line on this PR : https://github.com/rizinorg/rizin/pull/359.
I think this conversation can be resolved.